### PR TITLE
feat: add `vm.envUint` and `vm.setEnv` cheatcodes

### PR DESCRIPTION
**Pull request by aon**
# What :computer: 
* Implement `vm.envUint` cheatcode
* Implement `vm.setEnv` cheatcode

# Why :hand:
* To further support foundry on zksync-era

# Evidence :camera:
image

### CODE REVIEW

**Review by Deniallugo — APPROVED** [review]

File: crates/era-cheatcodes/tests/src/cheatcodes/Env.t.sol [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        string memory delimiter = ",";
+        uint256[] memory output = vm.envUint(key, delimiter);
+        require(
+            keccak256(abi.encodePacked((output))) ==
+                keccak256(abi.encodePacked((expected))),
+            "envUintArr failed"
+        );
+    }
+
+    function testEnvUintEmptyArr() public {
```
Comment (nbaztec) [inline crates/era-cheatcodes/tests/src/cheatcodes/Env.t.sol:57]: We could perhaps add a test case for non-empty array

**Review by nbaztec — APPROVED** [review]
